Makefile

```diff
@@ -8,3 +8,7 @@ gofmt:
 
 build-bin:
 	./build.sh
+
+test: build-bin
+	# Tests need sudo due to network interfaces creation
```

Comment:
test: build-bin # Tests need sudo due to network interfaces creation

Can you change to like this? So the comment does not print when run?

Reply:
fixed